### PR TITLE
#issue 235 ->Add Chinese translation

### PR DESCRIPTION
**Pull request by Skydrgon**
#235 
Add Chinese translation for README.md, the file is README-CN.md.
Updating help related to the nick and kick commands.

Not sure how to include a link to README-CN.md in README.md, do I need to commit this branch first and then reference it via markdown later?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ ssh wenjie@devzat.hackclub.com
 ```
 If you want to change your display name after the first login, you should use the `nick` command.
 
+```sh
+In Chat room:
+OLDnick> nick NEWnick
+NEWnick:
+```
+
```
Comment (quackduck) [inline README.md:36]: this doesn't deserve an example code block imo

File: README.md [hunk 2]
```diff
@@ -128,7 +134,7 @@ The rest
    pwd                     Show your current room
    shrug                   ¯\_(ツ)_/¯
 ```
-
+tips: `kick` can help kick out your previous id if the nickname is taken due to network delays.
```
[the remark this answers is not in the document]
Comment (quackduck) [reply README.md:137]: yeah good to include but imo a code block is not needed

**Comment by Skydrgon** [comment]
Took both of your comments and updated them.
Removed code block about nick.

File: README.md [hunk 3]
```diff
@@ -128,7 +129,7 @@ The rest
...
```
Comment (quackduck) [inline README.md:132]:
```suggestion
tip: `kick` can help kick out an old session when rejoining
```

**Comment by quackduck** [comment]
Looks good, thank you so much @Skydrgon!

**Pull request merged**